ui/src/components/chat/group_users.rs

```diff
@@ -42,6 +45,8 @@ pub fn GroupUsers(cx: Scope<Props>) -> Element {
             .map(|ident| (ident.did_key(), ident.clone())),
     );
     let _friends_in_group = State::get_friends_by_first_letter(hash_map);
+    let creator_id_vector = Vec::from_iter(active_chat.creator.iter().cloned());
+    let creator_id = creator_id_vector[0].clone();
```

Comment:
Might be better to do something like the following to prevent an index out of bounds panic
```suggestion
    let creator_id = creator_id_vector.get(0).cloned()?;
```